File: Beecrowd/3301 Sobrinho do Meio/main.py
h, z, l = [int(_) for _ in input().split()]

if (h > z and h < l) or (h < z and h > l):
    print("huguinho")
elif (z > h and z < l) or (z < h and z > l):
    print("zezinho")
else:
    print("luisinho")

File: Beecrowd/3301 Sobrinho do Meio/Main.java
import java.io.IOException;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);

        int h, z, l;

        h = sc.nextInt();
        z = sc.nextInt();
        l = sc.nextInt();

        if ((h > z && h < l) || (h < z && h > l))
            System.out.println("huguinho");
        else if ((z > h && z < l) || (z < h && z > l))
            System.out.println("zezinho");
        else
            System.out.println("luisinho");

        sc.close();
    }

}